services/ui-src/src/utils/api/requestMethods/banner.ts

```diff
@@ -1,40 +1,55 @@
-import { API } from "aws-amplify";
+import { get, post, del } from "aws-amplify/api";
 import { getRequestHeaders } from "./getRequestHeaders";
 import { AdminBannerData } from "types/banners";
 import { updateTimeout } from "utils";
 
+const apiName = "mcr";
+
 async function getBanner(bannerKey: string) {
   const requestHeaders = await getRequestHeaders();
-  const request = {
+  const options = {
     headers: { ...requestHeaders },
   };
+  const path = `/banners/${bannerKey}`;
 
   updateTimeout();
-  const response = await API.get("mcr", `/banners/${bannerKey}`, request);
-  return response;
+  const { body } = await get({
+    apiName,
+    path,
+    options,
+  }).response;
+  return (await body.json()) as unknown as { Item: AdminBannerData };
```

Comment:
Is casting to `unknown` necessary?

Reply:
the [build was failing](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/actions/runs/11280305608/job/31373091951) previously so Ben pushed [this change](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/pull/11876/commits/5ab08c6b4b9c9f0eb3d6e321cf6d6e3177a6636e) to resolve it. I too don’t love the casting but I need the build to pass. Open to alternatives!